Makefile

```diff
@@ -19,22 +17,11 @@ run-all-tests: run-nunit-tests
 
 # $(call RUN_NUNIT_TEST,filename,log-lref?)
 define RUN_NUNIT_TEST
-	MONO_TRACE_LISTENER=Console.Out \
-	$(RUNTIME) \
-		$(NUNIT_CONSOLE) $(NUNIT_EXTRA) $(1) \
-		$(if $(RUN),-run:$(RUN)) \
-		--result="TestResult-$(basename $(notdir $(1))).xml" \
-		-output=bin/Test$(CONFIGURATION)/TestOutput-$(basename $(notdir $(1))).txt \
-	|| true ; \
-	if [ -f "bin/Test$(CONFIGURATION)/TestOutput-$(basename $(notdir $(1))).txt" ] ; then \
-		cat bin/Test$(CONFIGURATION)/TestOutput-$(basename $(notdir $(1))).txt ; \
-	fi
+	dotnet test $(1) -l "console;verbosity=detailed"
 endef
 
-$(NUNIT_CONSOLE): prepare
-
 NUNIT_TESTS = \
-	bin/Test$(CONFIGURATION)/Xamarin.Android.Tools.AndroidSdk-Tests.dll
+	bin/Test$(CONFIGURATION)/netcoreapp3.1/Xamarin.Android.Tools.AndroidSdk-Tests.dll
```

Comment:
If `tests/Xamarin.Android.Tools.AndroidSdk-Tests/Xamarin.Android.Tools.AndroidSdk-Tests.csproj` isn't multi-targeted, could we set `$(AppendTargetFrameworkToOutputPath)` to `false`? Then this path wouldn't need to change.